examples/src/main/scala/com/example/http4s/ScienceExperiments.scala

```diff
@@ -103,21 +102,28 @@ object ScienceExperiments {
 
     ///////////////// Weird Route Failures //////////////////////
     case req @ GET -> Root / "hanging-body" =>
-      Ok(Process(Task.now(ByteVector(Seq(' '.toByte))), Task.async[ByteVector] { cb => /* hang */}).eval)
+      Ok(Stream.eval(Task.now(ByteVector(Seq(' '.toByte))))
+        .evalMap[Task, Task, Byte](_ => Task.async[Byte]{ cb => /* hang */}))
 
     case req @ GET -> Root / "broken-body" =>
-      Ok(Process(Task{"Hello "}) ++ Process(Task{sys.error("Boom!")}) ++ Process(Task{"world!"}))
+      Ok(Stream.eval(Task{"Hello "}) ++ Stream.eval(Task{sys.error("Boom!")}) ++ Stream.eval(Task{"world!"}))
 
-    case req @ GET -> Root / "slow-body" =>
-      val resp = "Hello world!".map(_.toString())
-      val body = awakeEvery(2.seconds).zipWith(Process.emitAll(resp))((_, c) => c)
-      Ok(body)
+//    Missing Strategy in Scope Which Is Clearly Not the Case
```

Comment:
It can't find the `Async[Task]`?  Yeah, what's up with that?  I've had to make that explicit a few times and am not understanding why.